src/fcntl.rs

```diff
@@ -504,3 +504,16 @@ mod posix_fadvise {
         Errno::result(res)
     }
 }
+
+#[cfg(any(
```

Comment:
You can add dragonfly, too.  You can also add NetBSD, if you make a PR to libc first.